src/main/webapp/app/exam/participate/exercises/text/text-exam-submission.component.scss

```diff
@@ -62,3 +62,21 @@ div {
         background-color: whitesmoke;
     }
 }
+
+#highlightDiffButton {
+    background-color: #fbc107;
+    border: none;
+    padding: 0px 10px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    margin-top: 0;
+    margin-bottom: 0;
+    margin-left: 10px;
+    font-size: 16px;
+    color: #353d47;
+
+    :hover {
+        color: #ffffff;
```

Comment:
not sure how to fix this one:
![image](https://user-images.githubusercontent.com/33342534/126346560-f56cee3d-3862-4564-8fe8-f9a83ed9a07b.png)

Reply:
```suggestion
        color: #fff;
```
I think it's just this